public/app/features/dashboard/components/DashboardSettings/PreviewSettings.tsx

```diff
@@ -50,6 +55,18 @@ export class PreviewSettings extends PureComponent<Props, State> {
             </tr>
           </thead>
           <tbody>
+            <tr>
+              <td>
+                <Button type="button" variant="primary" onClick={this.markAsStale(false)} fill="outline">
+                  Mark as stale
```

Comment:
Duplicate text on 2 buttons?

Reply:
Same text for two different themes ;) The current admin UI is very basic and won't be included in the mvp, I'll disable it in a separate PR https://github.com/grafana/grafana/issues/44449